src/hooks.server.ts

```diff
@@ -1,7 +1,14 @@
+import { dev } from '$app/environment';
 import { updateLocals } from '$lib/server/auth';
+import { sleep } from '$lib/util';
 import type { Handle } from '@sveltejs/kit';
+import { env } from '$env/dynamic/private';
 
 export const handle: Handle = async ({ event, resolve }) => {
+    // Add artificial delay when in dev so we can see what the web requests will be like
+    if (dev) {
+        await sleep(+(env.DELAY ?? 500));
+    }
```

Comment:
Not sure if I like this 500 ms as default in dev. This is lower than prod on my machine lol.

This does do a great job of illustrating how the optimistic actions work, but it's not super nice for just navigating the site.

Reply:
Yeah, that's fair.  I'll make it 0 by default.